Content/Items/Misc/Weapons.Earthduster.cs

```diff
@@ -206,7 +207,7 @@ public override void AI()
 						SoundEngine.PlaySound(SoundID.MaxMana with { Pitch = -0.5f }, Owner.Center);
 						if (Main.myPlayer == Projectile.owner)
 						{
-							Projectile proj = Projectile.NewProjectileDirect(Projectile.GetSource_FromThis(), armPos, Projectile.velocity * 1.5f, ModContent.ProjectileType<EarthdusterRing>(), 0, 0f, Owner.whoAmI, 15f);
+							var proj = Projectile.NewProjectileDirect(Projectile.GetSource_FromThis(), armPos, Projectile.velocity * 1.5f, ModContent.ProjectileType<EarthdusterRing>(), 0, 0f, Owner.whoAmI, 15f);
```

Comment:
I'm not a fan of var in general. it's very "javascripty" with the loose coupling. I like strongly typed languages that when you mess up the compiler will tell you exactly whats wrong. this adds that extra step of having to validate the type too when something goes wrong

Reply:
This change was made automatically by the formatter. If you want to propose a change to our formatting rules please do so in a seperate PR/Issue